pkg/api/v1/types.go

```diff
@@ -341,6 +363,9 @@ type MigrationSpec struct {
 type MigrationPhase string
 
 const (
+	// Create Migration has been called but nothing has been done with it
+	MigrationUnknown MigrationPhase = ""
```

Comment:
Is there a compelling reason to make this an empty string? It would be more consistent to make this:

+ MigrationUnknown MigrationPhase = "Unknown"